src/app/modules/auth/auth.component.html

```diff
@@ -1,11 +1,19 @@
 <nav class="navbar navbar-nav">
   <a class="logo"></a>
   <div class="actions">
-    <a href="" class="shopping-cart">
+    <a class="shopping-cart" (click)="openPopUp()">
       <span class="shopping-cart-image"></span>
       <span class="badge">{{count}}</span>
     </a>
     <a class="logout" (click)="logout()"><span class="logout-text">Logout</span></a>
   </div>
 </nav>
 <router-outlet></router-outlet>
+<app-pop-up popUpName="shopping" title="Tu carrito de compras" [open]="popUp" (closeAction)="closePopUp($event)">
```

Comment:
Create variable in component
```suggestion
<app-pop-up popUpName="shopping" [title]="title" [open]="popUp" (closeAction)="closePopUp($event)">
```